go/arrow/internal/flatbuf/DictionaryBatch.go

```diff
@@ -73,9 +73,9 @@ func (rcv *DictionaryBatch) Data(obj *RecordBatch) *RecordBatch {
 	return nil
 }
 
-/// If isDelta is true the values in the dictionary are to be appended to a
-/// dictionary with the indicated id. If isDelta is false this dictionary
-/// should replace the existing dictionary.
+// / If isDelta is true the values in the dictionary are to be appended to a
+// / dictionary with the indicated id. If isDelta is false this dictionary
+// / should replace the existing dictionary.
```

Comment:
I think you should undo these changes `git checkout -p origin/main` and have a separate PR where we make the who Go codebase auto-formattable.